crates/autopilot/src/run_loop.rs

```diff
@@ -268,6 +270,16 @@ impl RunLoop {
                 return;
             }
 
+            tracing::info!("saving fee policies");
+            if let Err(err) = self
+                .persistence
+                .store_fee_policies(auction_id, fee_policies)
+                .await
+            {
+                Metrics::fee_policies_store_error();
+                tracing::warn!(?err, "failed to save fee policies");
```

Comment:
If we cannot recover the fee policies later aren't we running into trouble from an accounting perspective? I think we should potentially not continue the run-loop in this case. Wdyt?

Reply:
I think @sunce86 mentioned that we should not stop the loop just because of the fee policies storing error since the information is not so important. Is this information outdated already?

Reply:
Autopilot/protocol is responsible for determining and storing/retrieving fee policies for orders. In that context, if it failed for some orders/auctions I believe it is ok to consider (in accounting) that we took 0 protocol fees for those orders/auctions.

I am in favor of keeping it as is (for now) since this is a new feature and there might be bugs. I would hate to have a prod downtime caused by protocol fee bug. Once stable, I would probably switch to more strict version (skip runloop).